src/cow-react/pages/NewSwap/index.tsx

```diff
@@ -9,38 +9,38 @@ import {
   useUnknownImpactWarning,
 } from 'state/swap/hooks'
 import { Field } from 'state/swap/actions'
-import { useSetupSwapState } from 'pages/NewSwap/hooks/useSetupSwapState'
+import { useSetupSwapState } from 'cow-react/swap/hooks/useSetupSwapState'
 import { useCurrencyBalance } from 'state/connection/hooks'
 import { useWeb3React } from '@web3-react/core'
-import { CurrencyInfo, SwapFormProps } from 'pages/NewSwap/typings'
+import { CurrencyInfo, SwapFormProps } from 'cow-react/pages/NewSwap/typings'
 import { useHigherUSDValue } from 'hooks/useStablecoinPrice'
-import { useSwapCurrenciesAmounts } from 'pages/NewSwap/hooks/useSwapCurrenciesAmounts'
+import { useSwapCurrenciesAmounts } from 'cow-react/swap/hooks/useSwapCurrenciesAmounts'
 import usePriceImpact from 'hooks/usePriceImpact'
 import { useWrapType, WrapType } from 'hooks/useWrapCallback'
-import { getInputReceiveAmountInfo, getOutputReceiveAmountInfo } from 'pages/NewSwap/helpers/tradeReceiveAmount'
-import { SwapButton } from 'pages/Swap/components/SwapButton/SwapButton'
-import { useSwapButtonContext } from 'pages/Swap/hooks/useSwapButtonContext'
+import { getInputReceiveAmountInfo, getOutputReceiveAmountInfo } from 'cow-react/swap/helpers/tradeReceiveAmount'
+import { SwapButton } from 'cow-react/swap/dumb/SwapButton/SwapButton'
+import { useSwapButtonContext } from 'cow-react/swap/hooks/useSwapButtonContext'
 import { useModalIsOpen } from 'state/application/hooks'
 import { ApplicationModal } from 'state/application/reducer'
-import { NewSwapModals, NewSwapModalsProps } from 'pages/NewSwap/modals'
-import { ConfirmSwapModalSetupProps } from 'pages/Swap/components/ConfirmSwapModalSetup'
+import { NewSwapModals, NewSwapModalsProps } from 'cow-react/swap/dumb/modals'
```

Comment:
Why `NewSwapModals` is not in its own directory?  why `modals`?

Reply:
Fixed